src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -156,8 +161,6 @@ private void Print(List<TestResult> testResults)
                 errorOutput = string.Format("Command: {0} {1}", _xunitConsolePath, builder.ToString())
                     + Environment.NewLine
                     + errorOutput;
-
-                Process.Start(resultsPath);
```

Comment:
@mmitche haven't thought about that yet.  I would hope for a similar one but I'd need to dig into the Process API a bit.